client/src/get-ticket/components/home.tsx

```diff
@@ -1,14 +1,45 @@
-import { useState } from 'react'
+import { useEffect } from 'react'
 import Button from 'react-bootstrap/Button';
 import '../get-ticket.css';
 import { useNavigate } from 'react-router-dom';
+import API from '../../API/API.tsx';
 
-function Home() {
+function Home(props: any) {
```

Comment:
Remember, In TypeScript always avoid "any" as much as you can.
Specially for props, never every use "any".
You need to provide which props you are refering specifically. That's the beauty of TS!
In this case, you may need to define it is the following:

props: Props 
(Props previsouly defined as an interface)